etl/churn.kp/knowledge.md

```diff
@@ -402,7 +409,9 @@ record_columns = [
     'is_active',
     'n_profiles',
     'usage_hours',
-    'sum_squared_usage_hours'
+    'sum_squared_usage_hours',
+    'total_uri_count',
```

Comment:
Please add the final field names and their description/interpretation in the Markdown blurb above (under "Compute the Aggregates").